cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -54,20 +57,34 @@ class OutletControllerTestIT extends ProjectStructureIT {
     private static final Location NEW_CONDUIT_GATE_1 = buildProjectStructureLocation(PROJECT_1_ID.getName() + "-CG10",
                                                                                      OUTLET_KIND);
     private static final Outlet NEW_CONDUIT_GATE_1_OUTLET = buildTestOutlet(NEW_CONDUIT_GATE_1, PROJECT_LOC,
-                                                                            CONDUIT_GATE_RATING_GROUP);
+                                                                            CONDUIT_GATE_RATING_GROUP, null);
 
     private static final Location EXISTING_CONDUIT_GATE = buildProjectStructureLocation(
             PROJECT_1_ID.getName() + "-CG20", OUTLET_KIND);
     private static final Outlet EXISTING_CONDUIT_GATE_OUTLET = buildTestOutlet(EXISTING_CONDUIT_GATE, PROJECT_LOC,
-                                                                               CONDUIT_GATE_RATING_GROUP);
+                                                                               CONDUIT_GATE_RATING_GROUP, null);
 
     private static final Location NEW_CONDUIT_GATE_2 = buildProjectStructureLocation(PROJECT_1_ID.getName() + "-CG30",
                                                                                      OUTLET_KIND);
     private static final Outlet NEW_CONDUIT_GATE_2_OUTLET = buildTestOutlet(NEW_CONDUIT_GATE_2, PROJECT_LOC,
-                                                                            CONDUIT_GATE_RATING_GROUP);
+                                                                            CONDUIT_GATE_RATING_GROUP, null);
     private static final CwmsId RENAMED_CONDUIT_GATE = new CwmsId.Builder().withName(
             NEW_CONDUIT_GATE_2.getName() + "_Renamed").withOfficeId(OFFICE_ID).build();
 
+    private static final String RATING_PROJECT_ID = "BIGH-TG1";
+    private static final String RATING_SPEC_ID = RATING_PROJECT_ID
+            + ".Opening-Low_Flow_Gates,Elev;Flow-Low_Flow_Gates.Standard.Production";
+    private static final CwmsId RATING_GROUP = new CwmsId.Builder().withName("Rating-" + RATING_PROJECT_ID
+            + "-RatedGate").withOfficeId(OFFICE_ID).build();
+    private static final Location RATED_OUTLET_LOCATION
+            = buildProjectStructureLocation(RATING_PROJECT_ID, OUTLET_KIND);
+    private static final Outlet NEW_RATED_OUTLET
+            = buildTestOutlet(RATED_OUTLET_LOCATION, PROJECT_LOC, RATING_GROUP, RATING_SPEC_ID);
+    private static final LocationGroup RATING_GROUP_LOCATION_GROUP
```

Comment:
This is not necessary for the tests, let the DB create the location group, and use the NEW_RATED_OUTLET's variables for retrieving the location group.  This will help solidify that the Outlet object is working as expected.

Reply:
Removed unnecessary LocationGroup creation, grabbed the object using the Outlet